app/src/pages/project/SessionsTable.tsx

```diff
@@ -46,12 +47,59 @@ type SessionsTableProps = {
 
 const PAGE_SIZE = 50;
 
+const TableBody = <T extends { id: string }>({
```

Comment:
Direct copy of the previous tbody content, minus new styles in the `td` element to prevent wrapping.